### PR TITLE
docs: fix greater than / less than comment

### PR DESCRIPTION
**Pull request by domodwyer**
Segments will always be less-than-or-equal-to the configured maximum
size, not greater-than-or-equal-to.

### CODE REVIEW

**Comment by domodwyer** [comment]
PR of the year!

File: data_types/src/database_rules.rs [hunk 1]
```diff
@@ -500,21 +500,21 @@ pub struct WriteBufferConfig {
...
-    /// happens if that segment hasn't been persisted. If set to return an
-    /// error, new writes will be rejected until the oldest segment has been
-    /// persisted so that it can be cleared from memory. Alternatively, this
-    /// can be set so that old segments are dropped even if they haven't been
-    /// persisted. This setting is also useful for cases where persistence
-    /// isn't being used and this is only for in-memory buffering.
+    /// Buffer size and the oldest segment that could be dropped hasn't yet been
+    /// persisted to object storage. If the oldest segment has been persisted,
+    /// then it will be dropped from the buffer so that new writes can be
+    /// accepted. This option is only for defining the behaviour of what happens
```
Comment (domodwyer) [inline data_types/src/database_rules.rs:511]: Fixes a typo spelling `behaviour` and reformats the block as a result.

Comment (e-dard) [reply data_types/src/database_rules.rs:511]: 🇬🇧 🇬🇧 🇬🇧 🇬🇧

**Review by e-dard — APPROVED** [review]
Bet you need a rest after this.

**Comment by domodwyer** [comment]
🍺

**Pull request merged**